tests/utils.go

```diff
@@ -4637,11 +4637,6 @@ func RandTmpDir() string {
 	return tmpPath + "/" + rand.String(10)
 }
 
-func IsIPv6Cluster(virtClient kubecli.KubevirtClient) bool {
```

Comment:
It makes sense to add the commit removing the method after there is no more code using this method:) But not critical to me as we are working in a PR scope.

Reply:
Thanks for explaining :)